src/scicookie/{{cookiecutter.project_slug}}/build-system/setup.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
```

Comment:
We are not adding type annotations anywhere in this file. What purpose is this import serving then?